src/main/java/eu/bbmri_eric/negotiator/dto/discoveryservice/GenericResource.java

```diff
@@ -0,0 +1,12 @@
+package eu.bbmri_eric.negotiator.dto.discoveryservice;
+
+public interface GenericResource {
```

Comment:
same as above

Reply:
Fixed in 3235f439a3ce3690a16b4781597181da46cebe37